packages/roosterjs-editor-plugins/lib/test/fragmentHandlerTest.ts

```diff
@@ -118,6 +118,26 @@ describe('fragmentHandler', () => {
                 );
             });
+            // e.g.
+            //text text
+            // .a
+            //   .b
+            //     1.c1
+            //     2.c2
+            //   .d
+            //text text
+            // -------------- --------------
+            //| text text    | text text    |
+            // -------------- --------------
+            //| .a           | .a           |
+            // -------------- --------------
+            it('fragments contains text, list and table that consist of list', () => {
+                runTest(
+                    '<html> <body> <div class="OutlineElement Ltr BCX0 SCXW244795937"> <div class="TableContainer SCXW244795937 BCX0"> <table> <tbody> <tr> <td> <div> <div class="OutlineElement Ltr BCX0 SCXW32709461"> <p><span><span>asdfasdf</span></span><span></span></p> </div> </div> </td> <td> <div> <div class="OutlineElement Ltr BCX0 SCXW32709461"> <p><span><span>asdfasdf222</span></span><span></span></p> </div> </div> </td> </tr> <tr> <td> <div> <div class="ListContainerWrapper SCXW244795937 BCX0"> <ul> <li role="listitem" data-aria-level="1" class="OutlineElement Ltr BCX0 SCXW244795937" style="margin: 0px 0px 0px 24px;"> A </li> </ul> </div> </div> </td> <td> <div> <div class="ListContainerWrapper SCXW244795937 BCX0"> <ul> <li role="listitem" data-aria-level="1" class="OutlineElement Ltr BCX0 SCXW244795937" style="margin: 0px 0px 0px 24px;"> A </li> </ul> </div> </div> </td> </tr> </tbody> </table> </div> </div> </body></html>',
+                    true,
+                    '  <div class="OutlineElement Ltr BCX0 SCXW244795937"> <div class="TableContainer SCXW244795937 BCX0"> <table> <tbody> <tr> <td> <div> <div class="OutlineElement Ltr BCX0 SCXW32709461"> <p><span><span>asdfasdf</span></span><span></span></p> </div> </div> </td> <td> <div> <div class="OutlineElement Ltr BCX0 SCXW32709461"> <p><span><span>asdfasdf222</span></span><span></span></p> </div> </div> </td> </tr> <tr> <td> <div> <div class="ListContainerWrapper SCXW244795937 BCX0"> <ul> <li role="listitem" data-aria-level="1" class="OutlineElement Ltr BCX0 SCXW244795937" style="margin: 0px 0px 0px 24px;"> A </li> </ul> </div> </div> </td> <td> <div> <div class="ListContainerWrapper SCXW244795937 BCX0"> <ul> <li role="listitem" data-aria-level="1" class="OutlineElement Ltr BCX0 SCXW244795937" style="margin: 0px 0px 0px 24px;"> A </li> </ul> </div> </div> </td> </tr> </tbody> </table> </div> </div> '

```

Comment:
Is this the minimized repro? We should keep the test case as small as possible. So that another day when we make some change breaks the case, we can understand why it is broken and how to fix.

Reply:
Just simplified the html removed some unecessary classnames and styles.